app/src/main/java/com/compscieddy/meetinthemiddle/adapter/ChatsAdapter.java

```diff
@@ -48,18 +47,16 @@ public int getItemCount() {
   public static final class ChatHolder extends RecyclerView.ViewHolder {
 
     @Bind(R.id.chat_icon) ImageView chatIcon;
-    @Bind(R.id.chat_message) TextView chatMessage;
+    TextView chatMessage;
 
     public ChatHolder(View itemView) {
       super(itemView);
-      ButterKnife.bind(itemView);
+      chatMessage = (TextView)itemView.findViewById(R.id.chat_message);
```

Comment:
You don't need to cast if you use ButterKnife.findById() @DarrenPoh this is what I was talking about.

`chatMessage = ButterKnife.findById(itemView, R.id.chat_message)`